tests/unit/js/test.subjects.js

```diff
@@ -0,0 +1,74 @@
+const sinon = require('sinon');
+const $ = require('../mockjQuery');
+window.$ = $;
+const subject = require('../../../openlibrary/plugins/openlibrary/js/subjects');
+
+beforeEach(() => {
+    window.$ = $;
+});
+
+afterEach(() => {
+    window.$ = undefined;
+    sinon.restore()
```

Comment:
Rather than having a beforeEAch and afterEach you might find it easier to setup a sandbox in beforeEach like so:

```
sandbox = sinon.sandbox.create();
sandbox.stub( global, '$', $ );
```

the use of a sandbox would save you needing to remember to teardown, which can be error prone if you forget something!